### PR TITLE
upgrading version numbers to 0.3

### PR DESCRIPTION
**Pull request by priesgo**

### CODE REVIEW

File: cva-storage/cva-storage-mongodb/pom.xml [hunk 1]
```diff
@@ -6,20 +6,20 @@
...
     <dependencies>
         <!-- Intra-module dependencies -->
         <dependency>
             <groupId>org.gel.cva</groupId>
             <artifactId>cva-storage-core</artifactId>
-            <version>0.2</version>
+            <version>0.3</version>
         </dependency>
     </dependencies>
 </project>
```
Comment (greglever) [inline cva-storage/cva-storage-mongodb/pom.xml]: I could be pedantic about there not being a new line here...

Comment (greglever) [inline cva-storage/cva-storage-mongodb/pom.xml]: :stuck_out_tongue_closed_eyes:

Comment (priesgo) [inline cva-storage/cva-storage-mongodb/pom.xml]: hahaha I didn't see this. I guess it is pedantic for your tone, but I don't understand what you mean :(

**Comment by greglever** [comment]
Just the lack of new line

**Pull request merged**